bundles/org.openhab.binding.mystrom/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -30,6 +45,54 @@
 
 	</thing-type>
 
+	<thing-type id="mystrombulb">
+		<label>myStrom Bulb</label>
+		<description>Controls the myStrom bulb</description>
+
+		<channels>
+			<channel id="switch" typeId="system.power"/>
+			<channel id="power" typeId="power-channel"/>
+			<channel id="color" typeId="system.color"/>
+			<channel id="colorTemperature" typeId="colorTemperature-channel"/>
```

Comment:
Please use the system color temperature channel type `system.color-temperature` for this and map your values 1-18 to 0-100% for it.